testing/plugins/plugins/main.tf

```
@@ -32,6 +32,7 @@ resource "leanspace_plugins" "test" {
   name                                 = "Terraform Command Transformer Plugin"
   description                          = "This is a plugin created through terraform!"
   source_code_file_download_authorized = true
+  sdk_version                          = "1.1.2"
```

```suggestion
  sdk_version                          = "2.1.2"
```
just so we try to use the newest version

---

I will update this in the next commit